### PR TITLE
Tokens: Adding Snapshot Tests for Token Dist

### PR DESCRIPTION
**Pull request by rlingineni**
This PR adds snapshot tests for the token outputs.

Because multiple platforms consume our token outputs, it's difficult to modify our `build.js` file safely without ensuring outputs aren't lost.

This adds snapshot tests and adds a check in CI. We can add more tests as needed

For example, these are the iOS outputs. If any of our work changes the outputs, or makes new ones, the snapshot test will have to be updated.

```
[ "dist/ios/GestaltDesignTokensColor.h",
  "dist/ios/GestaltDesignTokensColor.m",
  "dist/ios/GestaltDesignTokensColorDarkTheme.h",
  "dist/ios/GestaltDesignTokensColorDarkTheme.m",
  "dist/ios/classic/GestaltTokensColor.h",
  "dist/ios/classic/GestaltTokensColor.m",
  "dist/ios/classic/GestaltTokensColorDark.h",
  "dist/ios/classic/GestaltTokensColorDark.m",
  "dist/ios/classic/GestaltTokensElevation.
  ...
  ]
```

### CODE REVIEW

File: packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  "dist/css/variables.css",
+  "dist/css/vr/variables-dark.css",
+  "dist/css/vr/variables.css",
+  "dist/css/vr-theme/language/ck/line-height.css",
+  "dist/css/vr-theme/language/en/line-height.css",
+  "dist/css/vr-theme/line-height-ck.css",
+  "dist/css/vr-theme/line-height-en.css",
+  "dist/css/vr-theme/variables-dark.css",
+  "dist/css/vr-theme/variables.css",
+  "dist/css/vr-theme-web-mapping/language/ck/line-height.css",
```
Comment (AlbertCarreras) [inline packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap:45]: this doenst look it should be here heheh

File: packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  "dist/js/vr-theme/line-height-ck.css",
+  "dist/js/vr-theme/line-height-ck.js",
+  "dist/js/vr-theme/line-height-en.css",
+  "dist/js/vr-theme/line-height-en.js",
+  "dist/js/vr-theme/line-height-tokens.js",
+  "dist/js/vr-theme/tokens.js",
+  "dist/js/vr-theme/tokens_dark.js",
+  "dist/js/vr-theme-web-mapping/data-viz-tokens.js",
+  "dist/js/vr-theme-web-mapping/data-viz-tokens_dark.js",
+  "dist/js/vr-theme-web-mapping/line-height-ck.js",
```
Comment (AlbertCarreras) [inline packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap:105]: same here

**Review by AlbertCarreras — APPROVED** [review]
R= but some sems should be there.

Maybe clean the dist/ folder (remore folder + run yarn build)

File: packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap (hunk 1 above)
Comment (rlingineni) [reply packages/gestalt-design-tokens/tests/__snapshots__/dist.test.js.snap:45]: oh good catch - I think it's from my local build!

**Pull request merged**